src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const firstRejectionMessage = `First promise was rejected`;
+const firstResolvedMessage = `First promise was resolved`;
+const secondResolvedMessage = `Second promise was resolved`;
+const thirdResolvedMessage = `Third promise was resolved`;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (i) => {
+    if (i.button === 0) {
+      resolve(firstResolvedMessage);
+    }
+  });
```

Comment:
The event listener for the 'click' event in the first promise will not trigger if the 'setTimeout' function rejects the promise first. Consider whether this behavior is intended.